da/da_test.go

```diff
@@ -127,7 +117,9 @@ func doTestSubmitTimeout(t *testing.T, dalc *DAClient, blocks []*types.Block) {
 	require.NoError(t, err)
 
 	assert := assert.New(t)
+	defaultSubmitTimeout := submitTimeout
 	submitTimeout = 50 * time.Millisecond
+	defer func() { submitTimeout = defaultSubmitTimeout }()
```

Comment:
This can be a followup issue, but we need to stop using `submitTimeout` as global variable. The entire block in `da.go` should be changed from `var` to `const` and timeout should be stored with instance (const being just a default value).

Reply:
Added a constructor, which makes this easier.